libc/socket/unix-socket.c

```diff
@@ -930,6 +931,16 @@ void runner(void)
 
 int main(int argc, char *argv[])
 {
+	/* Due to scheduling delays of host system on which emulator runs,
+	 * add some extra value to poll timeout checks
+	 */
+	for (int i = 1; i < argc; i++) {
+		if (strcmp(argv[i], "--extra-poll-delay-ms") == 0) {
+			pollTimeoutDelay = atoi(argv[i + 1]);
```

Comment:
Should `pollTimeoutDelay` ever be allowed to have negative value? If not, maybe it'd be good to have a simple argument value validation here?

Reply:
It is passed directly in test.yaml, but I can add it.